### DISCUSSION TITLE
Endless 'Waiting for a runner to pick up this job...' for windows-latest

### DISCUSSION THREAD

**Discussion by 1kastner** [community/community#78802] [Actions]
### Select Topic Area

Bug

### Body

I have defined a CI pipeline that uses pip and conda to install a Python package once a week. This way, I am informed in case the installation procesdure fails. To be on the safe side, I even want to run the unittest test suite that is delivered alongside the package. The CI pipeline has worked fine for several months and now, out of the sudden, the windows runner gets stuck. The file is located at https://github.com/1kastner/conflowgen/blob/show-conflowgen-versions/.github/workflows/installation-from-remote.yaml and an example logs at https://github.com/1kastner/conflowgen/actions/runs/7179771823/job/19550672829?pr=201. The CI starts on windows-latest with cloning the repository, setting up the environment and installing the package. However, the last step of running the tests gets stuck. It shows this message:

```
2023-12-12T09:59:30.6527323Z Requested labels: windows-latest
2023-12-12T09:59:30.6527761Z Job defined at: 1kastner/conflowgen/.github/workflows/installation-from-remote.yaml@refs/pull/201/merge
2023-12-12T09:59:30.6527965Z Waiting for a runner to pick up this job...
2023-12-12T09:59:57.9704837Z Job is waiting for a hosted runner to come online.
2023-12-12T10:00:02.2104070Z Job is about to start running on the hosted runner: GitHub Actions 15 (hosted)
```

...and now more than 25 minutes have passed. This is repeatedly the case. I have also cancelled and restarted the workflow, to no avail.

**Comment by 1kastner** [comment]
Another odd thing to see - the CI works for Linux but not Windows. The code block containing `python -m pytest --pyargs conflowgen`  fails - also see https://github.com/1kastner/conflowgen/pull/203. I could not understand which part of the runner fails as there are no logs.

**Comment by swiknaba** [comment]
Removing blank lines in the YML file fixed the same issue for me. Sounds nuts, I didn't believe it, but it worked. For reference, also see this discussion https://github.com/orgs/community/discussions/31587#discussioncomment-8717992
